components/command/command-handler/src/main/java/uk/gov/justice/services/core/aggregate/AggregateService.java

```diff
@@ -66,7 +84,37 @@ private Object convertEnvelopeToEvent(final JsonEnvelope event) {
         if (!eventMap.containsKey(name)) {
             throw new IllegalStateException(format("No event class registered for events of type %s", name));
         }
-
         return jsonObjectToObjectConverter.convert(event.payloadAsJsonObject(), eventMap.get(name));
     }
-}
+
+    /**
+     * Sets object input stream strategy.
+     *
+     * @param objectInputStreamStrategy the object input stream strategy
+     */
+    public void setObjectInputStreamStrategy(final ObjectInputStreamStrategy objectInputStreamStrategy) {
```

Comment:
Yes used for only testing and now the visibility is reduced to package local instead of public, we dont have a need to inject it apart from testing.